app/models/article.rb

```diff
@@ -200,7 +202,7 @@ def interested_users
   end
 
   def notify_user_via_email(user)
-    EmailNotify.send_article(self, user) if user.notify_via_email?
+    # EmailNotify.send_article(self, user) if user.notify_via_email?
```

Comment:
Seems like you could leave in all the `EmailNotify` statements and just turn email off at a configuration level, no?